manuscript/manuscript.Rmd

```diff
@@ -491,68 +491,51 @@ Many of our most realistic ecological models exist only as numerical simulations
-As our examples have illustrated, it may be possible to gain intuition on an agent's behavior by visualizing the action selected against each possible state, though this becomes more difficult in large problems.
-Another challenge is that Deep RL algorithms are very sample inefficient, all though progress is being made to improve this [@cite]. <!-- ML: I think we can cut this sentence, mentionned it in passing earlier-->
--->
-
+Deep RL agents are also difficult to interpret [@heuillet_explainability_2021].
+Deep neural networks are commonly referred to as black box models, since it is challenging to fully explain why a deep neural network produces a specific output from a given input [@castelvecchi_can_2016].
+For deep RL, this means that we can rarely provide a meaningful explanation for why any agent selects an action.
+Compounded with the previously mentioned generalizability issues, the lack of explainability introduces a lot of safety concerns when attempting to apply these methods to the real world.
+When going from a simulator to the real world, the real world may introduce novel situations that a simulator-trained agent is not familiar with, and since we can not anticipate how an agent will respond to new situations, there is an inherent risk in letting RL agents make decisions in the real world. 
 
 <!-- reason 3: reward design -->
-<!-- ML: I'll take a pass at this paragraph in a new branch. I think it would be useful to put terms on stuff in here, like reward shaping and reward hacking which we are talking around. -->
-An issue that is central to any applied optimization problem is using the right objective.
-There have been many examples in which explicit reward design goes very wrong [cite].
-And for ecological systems, which are often very complex and impossible to fully observe, it seems that bad reward design is quite likely.
-Moreover, specifying a reward function for conservation brings up various ethical concerns -- e.g. who gets to specify the reward function and will the values of the greater community be represented in the reward function.
-RL researchers often introduce modified reward functions during training that help an agent perform better on the original objective [@cite].
-This approach is particularly common when rewards are sparse, such as in a game of Chess where the only objective that counts is whether or not you win, but during training, an RL algorithm may reward the agent for intermediate steps such as capturing a piece.
+And lastly, a fundamental issue for all applied optimization problems is using the right objective.
+There have been many examples in which explicit reward design produces unanticipated agent behavior that is highly undesirable [@hadfield-menell_inverse_2020].
+Thus, for ecological systems, which are typically quite difficult to control, specifying a reward function could easily produce behavior that opposes the intents of the RL practitioner.
```

Comment:
I think a concrete hypothetical would be useful here.  We haven't really discussed the idea of reward design as a strategy vs the reward really being the only thing you care about.  (e.g. if you're a fishing business and just trying to maximize profit, maybe you don't care about 'unexpected' solutions so long as it really is still maximizing the reward you care about.  The real problem comes along when the "RL" reward isn't the reward we care about, which isn't really a notion we have unpacked yet...